app/controllers/clients_controller.rb

```diff
@@ -9,5 +10,22 @@ def new
   end
 
   def create
+    client = Client.find_by(email: params[:client][:email])
+    if client
+      redirect_to new_client_path, notice: '登録済です'
```

Comment:
ここは早期returnで返してあげて、余計なネストをなくしてあげよう(plannerも同じく)

```suggestion
    return  redirect_to new_client_path, notice: '登録済です' if client
```

Reply:
確かにそうですね ありがとうございます